gui-doc/scribblings/gui/message-class.scrbl

```diff
@@ -65,5 +74,20 @@ The same as @xmethod[window<%> set-label] when @racket[label] is a
 Otherwise, sets the bitmap label for a bitmap message.
  @bitmaplabeluseisbm[label] @|bitmapiforiglabel|
 
-}}
+}
+
+@defmethod[(set-label-color [color color%]) void?]{
```

Comment:
This should be `(is-a?/c color%)`, right?

Reply:
Whoops, yes, thanks!